pkg/cmd/root/help.go

```diff
@@ -119,11 +119,18 @@ func rootHelpFunc(cs *iostreams.ColorScheme, command *cobra.Command, args []stri
 		Body  string
 	}
 
+	longText := command.Long
+	if longText == "" {
+		longText = command.Short
+	}
+	if longText != "" && command.LocalFlags().Lookup("jq") != nil {
```

Comment:
Seems like this might have meant to look up the `json` flag instead of the `jq` flag?

Reply:
Good catch! But I also wanted to apply this help text addition to the `gh api` command, which has `--jq` and `--template` but no `--json`.